CHANGELOG.md

```diff
@@ -15,16 +17,14 @@
 
 ## Packaging
 
-
-
 # v1.11.0
 
 ## Features
 
 - The `-L`/`--parameter-list` option can now be specified multiple times to
   evaluate all possible combinations of the listed parameters:
 
-  ``` bash
+  ```bash
```

Comment:
Do you have some kind of Markdown autoformatter, or why does the whole file have changes all over the place?

Reply:
Yes, I didn't even notice, my bad. Would you like me to revert the changes or are you ok with the formatting?

Reply:
it would be great if you could revert them.

Reply:
:+1: reverted